types/parse-decimal-number/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for parse-decimal-number 1.0
+// Project: https://github.com/AndreasPizsa/parse-decimal-number
+// Definitions by: Ciarán Ingle <https://github.com/ciaran1344>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type ArrayOptions = [thousands: string, decimal: string];
+
+interface HashOptions {
+    /**
+     * Decimal point character.
+     * Default: ".".
+     */
+    decimal: string;
+    /**
+     * Thousands separator character.
+     * Default: ",".
+     */
+    thousands: string;
+}
+
+type Options = ArrayOptions | HashOptions | string;
+
+declare const parseDecimalNumber: {
```

Comment:
This seems like a surprising way to declare this API; why not the more typical namespace + function declaration?